### ISSUE TITLE
INSTALL_GUIDE_LINUX - wrong descriptions

### ISSUE THREAD

**Issue by AlexImagineComm** [aws/aws-cdi-sdk#45]
1. Install EFA driver, part 2:
During step 3, part 5, install the minimum version of the EFA software
should be
During step 3, part **6**, install the minimum version of the EFA software
2. Install Package Dependencies
Ubuntu:
_sudo apt-get install –y libncurses-dev autoconf automake libtool libnl-3-dev_
The -y option does not work on ubuntu (should it be last in the command?).
3. Download AWS SDK
Verify that the necessary libraries are installed for AWS SDK for C++
Note: This procedure replaces these instructions: "Setting Up AWS SDK for C++". (It means they are not needed ???)
It's exactly the same link from the " libraries are installed for AWS SDK for C++" and ""Setting Up AWS SDK for C++". It is really hard to understand what should be verified and how. If you skip this step - the CDI could not be build at all. It took quite a time to understand that we really have to use this link: [Building-the-SDK-from-source-on-EC2](https://github.com/aws/aws-sdk-cpp/wiki/Building-the-SDK-from-source-on-EC2) and run the following commands:
$ sudo apt install g++ cmake -y
$ sudo apt install zlib1g-dev libssl-dev libcurl4-openssl-dev -y
I think this part should be clarified.

Regards

**Comment by AlexImagineComm** [comment]
4. One more question about the "Creating additional instances":
![image](https://user-images.githubusercontent.com/75698860/118026570-ecf0be80-b32e-11eb-84bd-3783db32cdda.png)
It seems quite a task to install and build everything again.  Isn't it easier to create AMI Image out of the Linux instance and create instance from this image ? This way the user should not install/build all the AWS CDI SDK/AWS CLI/CDI libs again. It seems to work with Windows Instances, which use CDI - would the same way work for the Linux as well ? Would it impede the functionalities of the EFA/CDI ?

**Comment by mhhen** [comment]
Alex,

Thank you for the detailed information. We reviewed the documentation and made several changes and included them as part of [Release 2.2.0](https://github.com/aws/aws-cdi-sdk/releases/tag/Release_2.2.0). Concerning your question about "Creating additional instances", creating an AMI would work too and does not impede EFA/CDI functionality. We will leave this ticket open until the documentation has been updated and made available.
